src/demo/App.vue

```diff
@@ -272,6 +276,12 @@ const pointsAxesLabelGridZoomAndLogScale = makeRandomPoints(pointPropsBasic);
 pointsAxesLabelGridZoomAndLogScale.forEach(p => p.x -= 0.5);
 const pointsPointsAxesAndZoom = makeRandomPoints(pointPropsBasic);
 const curvesTooltips = makeRandomCurves(propsBasic);
+const curvesArea = makeRandomCurves(propsBasic);
+curvesArea.forEach(c => {
+  c.fill = true;
+  c.style.fillColor = c.style.strokeColor;
+  c.style.fillOpacity = (c.style.opacity || 1) / 10;
+});
```

Comment:
I reckon these should be implied by an option within the props of makeRandomCurves - if we specify 'yes, fill please' in the props, then makeRandomCurves should set these three values.